mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/DebitCardFragment.java

```diff
@@ -95,7 +95,8 @@ public void verifyDebitCardSuccess(String otp) {
 
     @Override
     public void verifyDebitCardError(String message) {
-        showToast(message);
+
+        showToast(getString(R.string.debit_card_error_message));
```

Comment:
revert this change, please. `verifyDebitCardError` should be dynamic as it was, otherwise getting that string parameter doesn't make any sense.